seq_tools/validation/c609_fastq_sanity.py

```diff
@@ -116,9 +114,9 @@ def check(self):
 def fastq_test_length(fastq,path):
     file_path=os.path.join(path,fastq)
     if fastq.endswith("fastq.gz") or fastq.endswith("fq.gz"):
-        cmd="zcat "+file_path+" | wc -l"
+        cmd="cat "+file_path+" | zcat | wc -l"
```

Comment:
why to make these changes?

Reply:
This was a weird issue that came up during testing regarding mac using `zcat`:
https://serverfault.com/questions/570024/zcat-gzcat-works-in-linux-not-on-osx-general-linux-osx-compatibility